src/components/Button/Button.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Button.css';
+
+const Button = ({ htmlId, name, type, size = '', classname = '', onClick, children, ...props }) => (
+
+    <button
+        className={`btn mg-button ${size} ${classname}`}
```

Comment:
Have you considered using the [classnames](https://github.com/JedWatson/classnames) package?

Reply:
@mikestone14 It's a good idea, but I think for now it's not necessary. This is the most complicated the classnames will be, and only this component has something like this. I do love the classNames package though. i'll keep it in mind for the future